src/components/productView/productViewOpenShiftContainer.js

```diff
@@ -452,7 +452,10 @@ ProductViewOpenShiftContainer.defaultProps = {
 
             displayContent = translate('curiosity-graph.card-action-total', {
               context: 'coreHours',
-              total: numbro(total).format({ average: true, mantissa: 2, trimMantissa: true }).toUpperCase()
+              total:
+                (total > 999 &&
```

Comment:
optional: if there's a scenario where `999.XX` can be displayed this will attempt to apply numbro, recommend we switch to just `total >= 1000`